### ISSUE TITLE
Question about webpanel

### ISSUE THREAD

**Issue by Skvidd** [UnamSanctam/UnamWebPanel#396]
is webpanel mandatory when building cryptominer? And I tried to host a webpanel myself, but it isn't working 
![Screenshot_6](https://github.com/user-attachments/assets/6a2eedf5-c1ec-4046-bafb-76aba3fc246b)

**Comment by UnamSanctam** [comment]
No it's not mandatory, do you have PHP installed and enabled? And it's recommended to disable indexing on whatever webserver you're using.

**Comment by Skvidd** [comment]
yes I installed PHP and the problem still the same. Im hosting myself.

**Comment by UnamSanctam** [comment]
What are you using to host it? Apache, IIS or something like Nginx? And if you browse to the index.php file does that work? The reason that it shows the indexing could be because your webserver doesn't have index.php set as a default, which doesn't really matter.
